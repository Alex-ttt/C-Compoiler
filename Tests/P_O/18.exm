for (;;) ..
First action:
none
Stop condition:
none
Step:
none
Body:
  Statements:
    for (;;) ..
    First action:
    none
    Stop condition:
    none
    Step:
    none
    Body:
      Statements:
        for (;;) ..
        First action:
        none
        Stop condition:
        none
        Step:
        none
        Body:
          << block is empty >>
