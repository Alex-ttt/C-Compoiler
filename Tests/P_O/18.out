 Symbols Table:
   main  int function main(
  )
    {
      Statements:
      for (;;) ..
      First action:
      Stop condition:
      Step:
      Body:
        {
          Statements:
          for (;;) ..
          First action:
          Stop condition:
          Step:
          Body:
            {
              Statements:
              for (;;) ..
              First action:
              Stop condition:
              Step:
              Body:
                {
                  << block is empty >>
                }
            }
        }
    }
